substrate/frame/support/src/traits/try_runtime/decode_entire_state.rs

```diff
@@ -104,29 +117,38 @@ fn decode_storage_info<V: Decode>(
 		Some(bytes) => {
 			let len = bytes.len();
 			let _ = <V as DecodeAll>::decode_all(&mut bytes.as_ref()).map_err(|_| {
-				vec![TryDecodeEntireStorageError {
+				TryDecodeEntireStorageError {
 					key: key.to_vec(),
 					raw: Some(bytes.to_vec()),
 					info: info.clone(),
-				}]
+				}
 			})?;
 
-			Ok::<usize, Vec<_>>(len)
+			Ok::<usize, _>(len)
 		},
 	};
 
-	decoded += decode_key(&next_key)?;
```

Comment:
todo: not miss this key

Reply:
What you mean by this to do?

Reply:
I realised it currently skips the first key. Will fix today.

Reply:
fixed

Reply:
BTW, we did not miss the key. The first key being the `prefix` and we don't store there anything, but given how the code is written there should also be no problem, just one more db access.